ansible/roles/opencontrail-gateway/templates/ifcfg-gateway.j2

```diff
@@ -0,0 +1,3 @@
+DEVICE={{ item }}
+TYPE=vhost
+MACADDR=00:00:5e:00:01:00
```

Comment:
VRRP MAC address. These is a reserved mac address for a "virtual router". OpenContrail acts as a distributed virtual router so we present the same mac everywhere.